cmdtest.go

```diff
@@ -626,8 +626,9 @@ func expandVariables(s string, lookup func(string) (string, bool)) (string, erro
 // scrub removes dynamic content from output.
 func scrub(rootDir string, b []byte) []byte {
 	const scrubbedRootDir = "${ROOTDIR}"
-	rootDirWithSeparator := rootDir + string(filepath.Separator)
-	scrubbedRootDirWithSeparator := scrubbedRootDir + "/"
+	const sep = string(filepath.Separator)
```

Comment:
Can you use `filepath.Join` for these instead?

Reply:
I guess that wouldn't work, `filepath.Join` strips slashes: https://play.golang.org/p/sEoyjK_i5aW.